wordpress-mobile/WordPress-iOS
# -> User's email now EditableTextRow, enabling tap to change.

Fixes #4823 

To test: 

Launch app
Navigate to "me" tab and select "Account Settings"
Tap on e-mail row
You should now be able to edit your account's e-mail address.

Needs review: @astralbodies

-> Updated the property returned when generating an AccountSettingsChange of type Email.
-> Added localisation for the change email helper text. NOTE: Currently in English only.
